Samples.xcodeproj/project.pbxproj

```diff
@@ -1916,6 +1934,7 @@
 				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Not a big deal, pay attention to changes like this in the future. Sometimes easy-to-be-ignored minor changes may slip into a PR that causes unexpected side effect.